app/Settings/PterodactylSettings.php

```diff
@@ -6,11 +6,11 @@
 
 class PterodactylSettings extends Settings
 {
-    public string $admin_token;
+    public ?string $admin_token;
```

Comment:
Should the PterodactylSettings be nullable really? Without them controlpanel cannot work and will throw errors whenever accessing any page that depend on pterodactyl (like the overview, dashboard, server creation).

Reply:
I guess I shouldn't have committed that either then, i'll remove it.